app/grandchallenge/api/urls.py

```diff
@@ -74,6 +81,31 @@
     LandmarkAnnotationSetViewSet,
     basename="landmark-annotation",
 )
+router.register(
+    r"retina/image-level-annotation-for-image",
+    ImageLevelAnnotationsForImageViewSet,
+    basename="image-level-annotation-for-image",
```

Comment:
Should the base name be `retina-image-level-annotation-for-image`? Then it is more consistent with the other namespaces. But, these are so specific they're unlikely to clash...